main.go

```diff
@@ -43,11 +44,7 @@ import (
 )
 
 const (
-	flagMetricsAddr          = "metrics-addr"
-	flagEnableLeaderElection = "enable-leader-election"
-	flagLeaderElectionID     = "leader-election-id"
-	flagK8sClusterName       = "cluster-name"
-	defaultLeaderElectionID  = "aws-load-balancer-controller-leader"
+	defaultControllerPort = 9443
```

Comment:
seems this default value(defaultControllerPort) can be moved to runtime_config.go as well

Reply:
will include in the next commit.